Disabling contextual LOB creation as createClob() method threw error running on Linux with PostgreSQL db

Hello,

We are moving from the local Windows installation to the pre-prod one running on RedHat Linux entirely with our additional custom rules. 
The local Windows installation consists of the server (gradlew.bat) and web-ui image (with modified Docker file as suggested in one of the previous tickets) and run on in-memory HSQL db.
I've installed all pre-requisites including Docker and Yarn, etc., and somewhat successfully run ./build-and-run.sh shell script. 

It has created three images for me as follows:
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS                          PORTS                              NAMES
8742dcad1e52        zally_web-ui        "node server.js"         3 hours ago         Up About a minute               0.0.0.0:8080->3000/tcp             zally_web-ui_1
2a66aceb57ad        zally_server        "/bin/sh -c 'java ..."   3 hours ago         Up About a minute (unhealthy)   0.0.0.0:8000->8000/tcp, 8080/tcp   zally_server_1
75cf0f15310a        postgres:9.6        "docker-entrypoint..."   3 hours ago         Up About a minute               0.0.0.0:54321->5432/tcp            zally_postgres_1

where postgres:9.6 image is the one to be run in production. 

When the server was starting it brought the following exception:
.................................................................................................................
server_1    | 2019-06-18 15:19:11.635  INFO 11 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate Core {5.3.5.Final}
server_1    | 2019-06-18 15:19:11.637  INFO 11 --- [           main] org.hibernate.cfg.Environment            : HHH000206: hibernate.properties not found
server_1    | 2019-06-18 15:19:11.824  INFO 11 --- [           main] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.0.4.Final}
server_1    | 2019-06-18 15:19:12.213  INFO 11 --- [           main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.PostgreSQL95Dialect
server_1    | 2019-06-18 15:19:12.481  INFO 11 --- [           main] o.h.e.j.e.i.LobCreatorBuilderImpl        : HHH000424: Disabling contextual LOB creation as createClob() method threw error : java.lang.reflect.InvocationTargetException
server_1    | java.lang.reflect.InvocationTargetException: null
server_1    |   at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_191]
server_1    |   at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_191]
server_1    |   at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_191]
server_1    |   at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_191]
server_1    |   at org.hibernate.engine.jdbc.env.internal.LobCreatorBuilderImpl.useContextualLobCreation(LobCreatorBuilderImpl.java:113) [hibernate-core-5.3.5.Final.jar!/:5.3.5.Final]
.................................................................................................................
server_1    |   at org.springframework.boot.loader.Launcher.launch(Launcher.java:87) ~[zally.jar:na]
server_1    |   at org.springframework.boot.loader.Launcher.launch(Launcher.java:50) ~[zally.jar:na]
server_1    |   at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:51) ~[zally.jar:na]
server_1    | Caused by: java.sql.SQLFeatureNotSupportedException: Method org.postgresql.jdbc.PgConnection.createClob() is not yet implemented.
server_1    |   at org.postgresql.Driver.notImplemented(Driver.java:688) ~[postgresql-42.2.4.jar!/:42.2.4]
server_1    |   at org.postgresql.jdbc.PgConnection.createClob(PgConnection.java:1269) ~[postgresql-42.2.4.jar!/:42.2.4]
server_1    |   ... 52 common frames omitted

it looks like it is Hibernate issue and the quick search recommends to set this flag to 'true':
(https://github.com/spring-projects/spring-boot/issues/12007)

spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true

but I am not sure how to do this. The stack trace above says that Hibernate Properties not found.

The rest of the framework works perfect. I can see active rules as the linter service on port 8000. The whole home page works (on port 8080) with all links except it fails, obviously, on the rules validation.

Are there are any suggestion how fix PostgreSQL or just switch back to HSQL in-memory db?

Any help will be highly appreciated.

Thank you,
Vladimir Ivanov

--- a/server/src/main/resources/application.yml
+++ b/server/src/main/resources/application.yml
@@ -8,6 +8,14 @@ spring:
   jpa:
     hibernate:
       ddl-auto: validate
+    properties:
+      hibernate:
+        jdbc:
+          lob:
+            # Disable noisy detection for LOB support (Postgres doesn't support it anyway)
+            # Workaround from https://hibernate.atlassian.net/browse/HHH-12368
+            # Should be unnecessary after upgrading to Spring Boot 2.2.0
+            non_contextual_creation: true
   oauth2:
     resource:
       tokenInfoUri: ${TOKEN_INFO_URI}

